cves/kernel/CVE-2016-8650.yml

```diff
@@ -55,15 +55,19 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
```

Comment:
Replace "specificly" with "specifically". Replace your semicolon with a period. Put a comma between "memory" and "causing". Add punctuation.